pkg/gadgets/trace/network/types/types.go

```diff
@@ -41,7 +41,7 @@ type Event struct {
 	PodHostIP string            `json:"podHostIP,omitempty" column:"podhostip,template:ipaddr,hide"`
 	PodIP     string            `json:"podIP,omitempty" column:"podip,template:ipaddr,hide"`
 	PodOwner  string            `json:"podOwner,omitempty" column:"podowner,hide"`
-	PodLabels map[string]string `json:"podLabels,omitempty" column:"padlabels,hide"`
+	PodLabels map[string]string `json:"podLabels,omitempty" column:"podlabels,hide"`
```

Comment:
It seems this PR will take a bit more time. Meanwhile, I am fixing this typo in a separate PR:
https://github.com/inspektor-gadget/inspektor-gadget/pull/1339